backend/model/model.go

```diff
@@ -328,6 +328,7 @@ func (w *Workspace) AdminEmailAddresses(db *gorm.DB) ([]struct {
 			INNER JOIN admins a
 			ON wa.admin_id = a.id
 			WHERE wa.workspace_id = ?
+			AND wa.role = 'ADMIN'
```

Comment:
could we also audit that all sensitive emails we send use this `AdminEmailAddresses` list?

Reply:
@Vadman97 I checked other references to `workspace_admins` - I think this is the only sensitive email, the other ones are the digest and session insights emails, which should be fine to send out to everyone in the project, right?